meson_options.txt

```diff
@@ -0,0 +1 @@
+option('USE_GMP', type : 'boolean', value : true, description : 'Use gmp to support BigInt')
```

Comment:
https://mesonbuild.com/Build-options.html#features

Using a feature option instead of a boolean allows disabling it entirely. The current use will always check for this dependency, and setting the option to false just means that if it isn't found, don't error out.